habitat/core/dataset.py

```diff
@@ -35,6 +36,8 @@
 
 ALL_SCENES_MASK = "*"
 
+EpisodeInfo = namedtuple("EpisodeInfo", ["episode_id", "scene_id"])
```

Comment:
Can we instead just fetch all the episode meta data apart from `goals` json object in an episode? For some analysis on ObjectNav we usually need `object_category` and agent start state. I think the speed up comes because we just remove that huge goal view points json.

Reply:
Do we really want to a named a tuple here? Seems like an attr.s or dataclass would suite this better?

Reply:
@Skylion007 Incorporated your changes.

Reply:
@Ram81 where is this ObjectNav analysis codebase located?